infrastructure/lambda/api.yaml

```diff
@@ -195,22 +218,15 @@ paths:
         description: 'A valid access_token (e.g.: authorization: Bearer access-token-value).'
         schema:
           type: string
-    post:
-      summary: Post request using a valid access_token
-      description: Accepts form encoded param consisting of the access_token and subject
-
+    get:
+      summary: GET request using a valid access token
       responses:
         '200':
           description: 200 Ok
           content:
             application/jwt:
               schema:
-                type: "object"
-                properties:
-                  type: string
-                  format: application/jwt
-                  pattern: ^([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_\-\+\/=]+)$
-                  example: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
+                type: string
```

Comment:
IntelliJ wasn't happy with this block - I think we can just set the content type as application/jwt and body as string

Reply:
If the            ```     type: "object"
                properties:
```
is removed it would be fine, i think this got through during rebase and stuff

Reply:
@jkunle If I remove the `type: "object` then IntelliJ seems unhappy with the `properties` block